manipvortex/manipulator.go

```diff
@@ -697,40 +696,49 @@ func (m *vortexManipulator) monitor(ctx context.Context) {
 
 func (m *vortexManipulator) pushEvent(evt *elemental.Event) {
 
+	m.RLock()
+	defer m.RUnlock()
+
 	for _, s := range m.subscribers {
-		sevent, err := copystructure.Copy(evt)
-		if err != nil {
-			zap.L().Error("failed to copy event", zap.Error(err))
-			continue
-		}
 
-		if !s.filter.IsFilteredOut(evt.Identity, evt.Type) {
+		s.RLock()
+		ifFiltered := s.filter.IsFilteredOut(evt.Identity, evt.Type)
+		s.RUnlock()
+
+		if !ifFiltered && evt.Type == elemental.EventDelete && evt.Identity == "namespace" {
```

Comment:
My only concern here is if vortex is used elsewhere or for something else, I cannot filter the events so specifically.